templates/views/message-item.html

```diff
@@ -5,7 +5,11 @@
     <div class="ui segment {{avatar.color}}">
         <div class="summary">
             {{#if sender}}
+              {{#if disableSenderInfo}}
+                <p class="f-user">{{senderName}}</p>
+              {{else}}
```

Comment:
NIT: spacing is inconsistent.  4 spaces are being used in surrounding context.

Reply:
Yeah, my VS code is set to two spaces. Don't always catch it